modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/semanticui/elements/button/Button.scala

```diff
@@ -0,0 +1,70 @@
+package edu.gemini.seqexec.web.client.semanticui.elements.button
+
+import japgolly.scalajs.react.{Callback, ReactComponentB, ReactNode}
+import japgolly.scalajs.react.vdom.prefix_<^._
+
+object Button {
+  sealed trait ButtonState
+  case object Active extends ButtonState
+  case object Inactive extends ButtonState
+
+  sealed trait Emphasis
+  case object NoEmphasis extends Emphasis
+  case object Primary extends Emphasis
+  case object Secondary extends Emphasis
+
+  sealed trait Animated
+  case object NotAnimated extends Animated
+  case object Animated extends Animated
+  case object Vertical extends Animated
+  case object Fade extends Animated
+
+  case class Props(state: ButtonState = Inactive,
+                   emphasis: Emphasis = NoEmphasis,
+                   animated: Animated = NotAnimated,
+                   icon: Boolean = false,
+                   basic: Boolean = false,
+                   inverted: Boolean = false,
+                   circular: Boolean = false,
+                   tabIndex: Option[Int] = None,
+                   color: Option[String] = None,
+                   onClick: Callback = Callback.empty)
+
+  def classSet(p: Props) =
+    ^.classSet(
+      "active"    -> (p.state == Active),
+      "primary"   -> (p.emphasis == Primary),
+      "secondary" -> (p.emphasis == Secondary),
+      "animated"  -> (p.animated != NotAnimated),
+      "vertical"  -> (p.animated == Vertical),
+      "fade"      -> (p.animated == Fade),
+      "icon"      -> p.icon,
+      "basic"     -> p.basic,
+      "inverted"  -> p.inverted,
+      "circular"  -> p.circular
+    )
+  def component = ReactComponentB[Props]("Button")
+    .renderPC((_, p, c) =>
+      if (p.animated == NotAnimated)
+        <.button(
+          ^.cls := "ui button",
+          p.color.map(u => ^.cls := u),
+          ^.tabIndex := p.tabIndex,
```

Comment:
It is similar to the tab index of a swing component. It is required in this case because otherwise you cannot click into a button with just the keyboard.